WDE.DatabaseEditors/Solution/DbEditorsSolutionItem.cs

```diff
@@ -0,0 +1,23 @@
+﻿using System.Collections.ObjectModel;
+using WDE.Common;
+using WDE.DatabaseEditors.Models;
+
+namespace WDE.DatabaseEditors.Solution
+{
+    public class DbEditorsSolutionItem : ISolutionItem
+    {
+        public DbEditorsSolutionItem(DbTableData tableData)
+        {
+            TableData = tableData;
+            Items = null;
+            ExtraId = TableData.TableIndexValue;
+        }
+
+        public DbTableData TableData { get; }
```

Comment:
You cannot keep all those data here. Most of those things shouldn't be serialized. Only name of the table, key and fields should be serialized.

Not other things like type or "key name" for instance.